### PR TITLE
region_requst: add req info in flashback region error

### PR DESCRIPTION
**Pull request by Defined2014**
add more log for flashback when it meets error, ref https://github.com/pingcap/tidb/issues/43673

### CODE REVIEW

File: internal/locate/region_request.go [hunk 1]
```diff
@@ -1752,6 +1752,9 @@ func (s *RegionRequestSender) onRegionError(
 				return true, nil
 			}
 		}
+		logutil.BgLogger().Error("Region is in flashback progress",
+			zap.Stringer("req", req),
```
Comment (you06) [inline internal/locate/region_request.go:1756]: I'm inclined to contain these info inside the returned error, also consider the req can be nil.

Also, `req.String()` return weird things that no meaningful information is inside it. Maybe we should format some key fields of it.

**Comment by you06** [comment]
@Defined2014 I suggest to attach the addtional info like this:
https://github.com/tikv/client-go/compare/master...you06:client-go:add-log?expand=1

**Comment by Defined2014** [comment]
> @Defined2014 I suggest to attach the addtional info like this: https://github.com/tikv/client-go/compare/master...you06:client-go:add-log?expand=1

ok. I found that we already have debug log in this branch, maybe we could try open debug log firstly.

And not sure return such internal information in Error Msg is good or not. It's useless for user.

**Comment by you06** [comment]
> ok. I found that we already have debug log in for this part, maybe we could try open debug log firstly.

Debug mode print too many logs...

File: internal/locate/region_request.go (hunk 1 above)
Comment (Defined2014) [reply internal/locate/region_request.go]: Updated.

**Pull request closed without merging**